mff_rams_plugin/templates/groupextra.html

```diff
@@ -13,6 +13,7 @@
     };
     {% endif %}
     $(function () {
+        $("#table_prices").hide(); // never show the table prices, since they duplicate the dropdown and the fn is wrong
```

Comment:
This might break JS on certain pages as-is, let's put it inside a function that checks for the existence of the div to be safe.